packages/ui-stencil/src/components/orama-chat-box/orama-chat-box.tsx

```diff
@@ -53,7 +54,12 @@ export class ChatBox {
           sourceBaseUrl={this.sourceBaseUrl}
           sourcesMap={this.sourcesMap}
           suggestions={this.suggestions}
-        />
+          focusInput={this.autoFocus}
+        >
+          <div slot="empty-state">
+            <slot name="empty-state" />
+          </div>
```

Comment:
```suggestion
          <slot name="empty-state" />
```

Reply:
I tried this way but it's not working, as we need to place the custom empty state content within the `orama-chat `child component within the dedicated slot area (see `orama-chat`)
I initially did something like:
`<slot name="empty-state" slot="chat-empty-state"/>` 
It was actually working but seems to be not a standard orthodox approach, so I removed it :')